src/test/resources/tags/extendstag/tagcycleexception/library-template.jinja

```diff
@@ -0,0 +1,2 @@
+{{ variable1 }}
```

Comment:
are these variables relevant to the test?

Reply:
No, you're right, we don't really need them here. 
Removed them.